ts/WoltLabSuite/Core/Notification/Handler.ts

```diff
@@ -90,18 +99,18 @@ class NotificationHandler {
    */
   private getNextDelay(): number {
     if (this.inactiveSince === 0) {
-      return 5;
+      return 1;
     }
 
     // milliseconds -> minutes
     const inactiveMinutes = ~~((Date.now() - this.inactiveSince) / 60_000);
     if (inactiveMinutes < 15) {
-      return 5;
+      return 1;
     } else if (inactiveMinutes < 30) {
-      return 10;
+      return 1;
     }
 
-    return 15;
+    return 1;
```

Comment:
What is the purpose of this change?

Reply:
This should not be changed and is for testing purposes only. Was a wrong commit.